### ISSUE TITLE
Unable to use EnFr Pretrained Model

### ISSUE THREAD

**Issue by NehaPathapati** [tensorflow/tensor2tensor#824]
### Description

> I see that there is a pretrained model for English to French named 'transformer_enfr_test_1201' on Google Storage. I am unable to get it working with the translate_enfr_wmt32k problem.

For all input sentences, I get the following French translation:
attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted attracted

### *TensorFlow* and *tensor2tensor* versions

> …

### In case of bug report: Steps to reproduce the problem

> …

### In case of bug report: Error log

> …

**Comment by martinpopel** [comment]
Make sure you use the correct vocabulary file (should be stored somewhere near the pretrained model on Google Storage).

**Comment by NehaPathapati** [comment]
This is the vocab file I am using right now: gs://tensor2tensor-data/vocab.enfr.32768
Problem: translate_enfr_wmt32k
Model: transformer
hparams_set: transformer_base

Are any of these not right?

**Comment by NehaPathapati** [comment]
So there is no pre-trained model for English to French now?

**Comment by rsepassi** [comment]
correct. we did not have (a good) one before and there is not one now.
